client-guide/jmap-client-guide.mdown

```diff
@@ -519,38 +515,46 @@ Let's look at a common case example, where two new messages have been delivered
         "list": [{
             "id": "fm1u316",
             "threadId": "afed75fb4f512ae7",
-            "mailboxId": "mailbox1",
-            "isUnread": true,
-            "isFlagged": false,
-            "isAnswered": false,
-            "isDraft": false,
+            "mailboxIds": {
+                "mailbox1": true
+            },
+            "keywords": {
+                "$seen": false,
+                "$flagged": false,
+                "$answered": false,
+                "$draft": false
```

Comment:
Same.

Reply:
For this one the scenario is that two new emails were received, so I think it should be `keywords: {}`